packages/server/src/simulation.ts

```diff
@@ -62,3 +61,19 @@ export function simulation(definitions: Record<string, Simulator>): Effect<Simul
     }
   };
 }
+
+function selectBehaviors(simulators: Record<string, Simulator>, selected: string[]): Behaviors {
+  return selected.reduce((behaviors, selection) => {
+    assert(simulators[selection] != null, `unknown simulator ${selection}`);
```

Comment:
small thing and this can be set in eslint to disallow `==` and `!=` and favour the strict `===` and `!=='.

So rather than coalese `null` to `undefined`, is it not better to say `typeof simulators[selection]` or `!!simulators[selection]`?

if you think it reads better then that is fine also.

Reply:
I think it's muscle memory from my CoffeeScript days